bin.js

```diff
@@ -2,8 +2,7 @@
 
 var proc = require('child_process')
 
-proc.exec('node-gyp-build-test', function (err) {
+proc.exec('prebuildify-load-test', function (err) {
   if (err) process.exit(1)
```

Comment:
insert console.log(err) here and run prebuildify-load. You'll get a command not found error every time. if you like I can split this PR into two. I'm pretty sure there is no doubt with this commit that this is a simple typo.

Reply:
dang, you're absolutely right! how did this ever work

Reply:
I published this change as 2.0.0